Geometry/HcalTowerAlgo/test/HcalGeometryAnalyzer.cc

```diff
@@ -19,62 +22,65 @@ class HcalGeometryAnalyzer : public edm::one::EDAnalyzer<>
   void endJob() override {}
 
 private:
-  std::string m_label;
+  edm::ParameterSet ps0;
```

Comment:
@bsunanda - please, use consistent naming of private members. For example, I use "m_name". It would be fine to use "name_" as well.